editor/blocks/text-block/index.js

```diff
@@ -1,11 +1,11 @@
-const { query, html } = wp.blocks.query;
+const { html } = wp.blocks.query;
 
 wp.blocks.registerBlock( 'wp/text', {
 	title: 'Text',
 	icon: 'text',
 
 	attributes: {
-		value: query( 'p', html() )
+		value: html()
```

Comment:
I'm assuming we're talking about the visual editor. Then yes, in its current form this block saves the content with the `p` tag. A block declares its saved markup in the `save` function. You can see that the current `wp/text` block wraps the content with a `p` tag.

But this is far from being the final version of the text block, it's just here to serve as an example of block while we're building the "surroundings", how we parse the content, how we save it back, how we render the editables etc...